dashboard/app/views/layouts/_header.html.haml

```diff
@@ -98,7 +98,7 @@
                 = render file: Rails.root.join('..', 'shared', 'haml', 'hamburger.haml'), locals: hamburger_options
 
 - if show_header2
-  - if request.path == "/" || request.path == "/courses"
+  - if request.path == "/home" || request.path == "/courses"
```

Comment:
I see. Yeah, that's not better. As far as page properties, AFAIK Dashboard's equivalent is set up by controllers in `view_options`, which can be inspected by other views. The views shouldn't need to declare anything that will be consumed outside of their scope.  Also FWIW I find these Pegasus page properties difficult to trace, though I may just be unfamiliar.